ui/public/i18n/language.json

```diff
@@ -429,7 +429,10 @@
         "udpPorts": "UDP ports",
         "access": "Access",
         "service_edited_successfully": "Service edited successfully",
-        "service_edited_error": "Service not edited"
+        "service_edited_error": "Service not edited",
+        "order": "Rule position",
+        "Create_first_label": "Create at the top",
```

Comment:
`"Create_first_label": "Create at the top"` the json attribute should be in lower case:
`"create_first_label": "Create at the top"`

Reply:
done